### DISCUSSION TITLE
Only using 1 core despite settings saying 37?

### DISCUSSION THREAD

**Discussion by Cigoler** [WinMerge/winmerge#2351] [Q&A] [answered]
Just comparing a network share folder to a local drive folder and it says it's only using 1 core. Just curious if I'm doing something wrong or if that's a limitation?

**Comment by sdottaka** [comment, marked as answer]
What folder compare method are you using?
If you choose anything other than Full contents or Quick contents, WinMerge uses only one CPU core for the comparison, because using multiple CPU cores can slow things down.

![image](https://github.com/WinMerge/winmerge/assets/98126/723bac02-3a15-45b4-ab18-0fb79e0056d6)

**Reply by Cigoler** [reply]
Yeah that's exactly what I was doing thanks, shame multiple cores can't speed up the simpler compare method but I'm not the one programming it.
